libnetdata/ebpf/ebpf.h

```diff
@@ -87,6 +88,14 @@
 #define VERSION_STRING_LEN 256
 #define EBPF_KERNEL_REJECT_LIST_FILE "ebpf_kernel_reject_list.txt"
 
+
+typedef struct ebpf_addresses {
+    char *function;
+    uint32_t hash;
+    // We use long as address, because it matches system length
+    unsigned long addr;
```

Comment:
For `ext4` we do not use the function that stores the data, but when we bring `btrfs` you will see that inside the kernel we compare direct `unsigned long` values, so my choice was to simplify the developer read.

On user ring, this will be always a number, we won't use it to do any comparison on user ring.